docs/src/pages/guides/aliases.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Aliases
+---
+
+Aliasing is an integral part when it comes to clean and readable code, that's why Astro comes with support for custom aliases out of the box.
+
+## What are aliases?
+
+Aliases are _keywords_ you can use instead of paths. 
```

Comment:
consider removing this header, since the start of every page gets an automatic "Overview" header which these two lines feel like they could fall under.